video/out/drm_common.c

```diff
@@ -588,6 +628,84 @@ static bool setup_crtc(struct vo_drm_state *drm, const drmModeRes *res)
     return true;
 }
 
+static void setup_edid(struct vo_drm_state *drm)
+{
+    drmModePropertyBlobRes *blob;
```

Comment:
This needs to be initialized for the loop below to work correctly. (you can also reorder loop body to avoid need to init)

Reply:
I made explicitly made it NULL. Not sure how it could be reordered since I need to use the blob later?